pkg/controller/eventing/subscription/reconcile.go

```diff
@@ -167,17 +167,17 @@ func (r *reconciler) updateStatus(subscription *v1alpha1.Subscription) (*v1alpha
 }
```

Comment:
this file has one more reference to `Targetable`: 

https://github.com/knative/eventing/blob/331bf4915aeace288f747758b7504272bacb578b/pkg/controller/eventing/subscription/reconcile.go#L106